_docs/services/aws-elasticsearch.md

```diff
@@ -51,6 +51,9 @@ cf create-service aws-elasticsearch es-medium my-elastic-service
 
 Note: AWS Elasticsearch creation times will vary and is outside of Cloud.gov's control. AWS says approximately 15-30 mins per node.
 
+## Changing instance plans
+Please note that you can only update to a larger plan size. Downgrading to a smaller plan size is not supported.
```

Comment:
```suggestion
Please note that you cannot use the broker to update the instance plan at this time.
```

It looks like we don't support changing the plan regardless of whether it's a bigger or smaller plan: https://github.com/cloud-gov/aws-broker/blob/main/services/elasticsearch/broker.go#L183